app/string.js

```diff
@@ -0,0 +1,52 @@
+String.prototype.hasVowels = function(){
+    return /[aeiou]/i.test(this);
+  };
+String.prototype.toUpper = function() {
+    return this.replace(/[a-z]/g, (lowerChar) => {
+      return String.fromCharCode(lowerChar.charCodeAt(0)-32);
+    });
+  };
+String.prototype.toLower = function() {
+    return this.replace(/[A-Z]/g, (upperChar) => {
+      return String.fromCharCode(upperChar.charCodeAt(0)+32);
+    });
+};
+String.prototype.ucFirst = function() {
+    return this.replace(/[a-z]/, (upperFirst) => {
+      return upperFirst.toUpper();
+    });
+};
+String.prototype.isQuestion = function() {
+    return /[?]/.test(this);
+};
+String.prototype.words = function() {
+    var uWords = [];
+    var wordArray = this.replace(/\W+/g, ' ').trim().split(' ');
+    wordArray.forEach(function(word){
+      if(uWords.indexOf(word) === -1){
+        uWords.push(word);
+      }
+    });
+    return uWords;
+};
+
+String.prototype.wordCount = function() {
+    return this.words().length;
+};
+/*
+split the string from the decimal point then look for
+a pattern in the first element of the array.
+*/
+String.prototype.toCurrency = function() {
```

Comment:
Have you considered cases where the user doesn't provide a decimal.
e.g. `'462391729750'.toCurrency()` returns `"462,391,729,750."`
It has a hanging decimal point at the end.